Tests/AppStoreServerLibraryTests/UtilityTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  UtilityTests.swift
+//  AppStoreServerLibrary
+//
+//  Created by shimastripe on 2025/11/17.
+//
+
+import XCTest
+@testable import AppStoreServerLibrary
+
+final class Base64URLToBase64Tests: XCTestCase {
```

Comment:
Is this not already tested by the ChainVerifier tests now that TestingUtility is corrected? Could we stick with those tests please, if so

Reply:
@shimastripe Could we please remove this test now and break this out into a separate PR for separate consideration so we can go ahead and get this fix merged?